.generator/templates/model/model.mustache

```diff
@@ -38,7 +38,7 @@ export class {{classname}} {{#parent}}extends {{{.}}} {{/parent}}{
         {{/vars}}
     };
 
-    static getAttributeTypeMap() {
+    static getAttributeTypeMap(): {[key: string]: {baseName: string, type: string, required?: boolean, format?: string}} {
```

Comment:
could we create a new type alias for `{[key: string]: {baseName: string, type: string, required?: boolean, format?: string}}`?

Reply:
Done!